prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,32 @@
+const number1 = 5;
+const number2 = 5;
+const product = number1 * number2;
+console.log(product);
+console.log(typeof product);
+const payment = 5;
+const charge = 8;
+const amountRemaing = payment - charge;
+console.log(amountRemaing);
+console.log(typeof amountRemaing);
+const tests = 6;
+const assignments = 11;
+const final = 7;
+const grade = tests + assignments + final;
+console.log(grade);
+console.log(typeof grade);
+const firstName = `Andrew`;
+const lastName = `Ortiz`;
+const fullName = `${firstName}${lastName}`;
+console.log(fullName);
+console.log(typeof fullName);
+const pH = 7;
+const isAcidic = pH < 5;
+console.log(isAcidic);
+console.log(typeof isAcidic);
+const numberOfSoldiers = 300;
+const isSparta = 1;
+console.log(numberOfSoldiers === isSparta);
```

Comment:
you want to save the result of comparing the number of soldiers to 300 into the `isSparta` variable and then log that variable